ci/tests.sh

```diff
@@ -4,4 +4,7 @@ set -e
 # -----
 # REPLACE THE BELOW WITH YOUR COMMANDS
 # -----
-echo 'Run tests'
\ No newline at end of file
+npm ci
+npx playwright install --with-deps
```

Comment:
```suggestion
npx playwright install
```

I don't know if this hurts, but tried removing the --with-deps and it worked OK.

Reply:
Okay, when I did it originally, it didn't work but I the `html` to `line` change could have been what was causing the hang up. Going to remove the deps and see what it does.